app/views/services_and_information/_grouped_links.html.erb

```diff
@@ -13,7 +13,7 @@
       <%
         list_item_count = group.see_more_link ? group.examples.count + 1 : group.examples.count
         tracking_attributes = topic_list_tracking_attributes(list_item_count, group_index, 'navServicesInformationLinkClicked')
-        topic_list_component_params = topic_list_params(group.examples, tracking_attributes: tracking_attributes)
+        topic_list_component_params = topic_list_params(group.examples, list_index: group_index, tracking_attributes: tracking_attributes, list_count: grouped_links.count, list_title: group.title)
```

Comment:
Getting quite a number of params here, not necessarily this time - but maybe a small refactor next time we need to add any.